raiden/ui/cli.py

```diff
@@ -424,27 +422,28 @@ def options(func):
                 "--flat-fee",
```

Comment:
Christian proposed to rename these to `XXX_fee_for_token`. What do you think?

Reply:
I don't see why the `_for_token` suffix is needed. Make all of the arguments, some of which are already pretty long/verbose even longer. So I would not do that.

In any case, any such cosmetic changes should be done after this PR. It has a lot of commits and the longer it's kept open the harder the more rebases it has to go through.